package/MDAnalysis/analysis/pca.py

```diff
@@ -181,46 +181,39 @@ def __init__(self, universe, select='all', align=False, mean=None,
         n_components : int, optional
             The number of principal components to be saved, default saves
             all principal components, Default: None
-        start : int, optional
-            First frame of trajectory to use for generation
-            of covariance matrix, Default: None
-        stop : int, optional
-            Last frame of trajectory to use for generation
-            of covariance matrix, Default: None
-        step : int, optional
-            Step between frames of trajectory to use for generation
-            of covariance matrix, Default: None
         verbose : bool (optional)
              Show detailed progress of the calculation if set to ``True``; the
              default is ``False``.
         """
-        super(PCA, self).__init__(universe.trajectory,
-                                  **kwargs)
-        if self.n_frames == 1:
-            raise ValueError('No covariance information can be gathered from a'
-                             'single trajectory frame.\n')
-
+        super(PCA, self).__init__(universe.trajectory, **kwargs)
         self._u = universe
 
         # for transform function
         self.align = align
-        # access 0th index
-        self._u.trajectory[0]
-        # reference will be 0th index
-        self._reference = self._u.select_atoms(select)
-        self._atoms = self._u.select_atoms(select)
+
+        self._calculated = False
         self.n_components = n_components
+        self._select = select
+        self._mean = mean
+
+    def _prepare(self):
+        # access start index
+        self._u.trajectory[self.start]
```

Comment:
I'm not sure if this is right.  Previously this used the 0th frame, but it makes more sense to use the `self.start` frame right?

Reply:
Yes, it makes more sense to me to use `start`. Should be documented, though.